demos/speech_recognition_demo/python/speech_recognition_demo.py

```diff
@@ -55,56 +65,74 @@ def get_profile(profile_name):
 
 
 def main():
-    start_time = timeit.default_timer()
-    with Timer() as timer:
-        args = build_argparser().parse_args()
-        profile = get_profile(args.profile)
+    args = build_argparser().parse_args()
+    profile = get_profile(args.profile)
+    if args.block_size is None:
+        sr = profile['model_sampling_rate']
+        args.block_size = round(sr*10) if not args.online else round(sr*profile['frame_stride_seconds']*16)
 
-        stt = DeepSpeechPipeline(
+    with Timer() as timer:
```

Comment:
Since this is only one usage of `Timer` class, it's the best time to delete it.

Reply:
Done. Removed Timer class.